src/Build.OM.UnitTests/Microsoft.Build.Engine.OM.UnitTests.csproj

```diff
@@ -55,6 +55,9 @@
     <Compile Include="..\Shared\ResourceUtilities.cs">
       <ExcludeFromStyleCop>true</ExcludeFromStyleCop>
     </Compile>
+    <Compile Include="..\Shared\Traits.cs">
+      <Link>Traits</Link>
```

Comment:
I don't know, but it makes much more sense to call it `SharedTraits.cs` so I did. Nice catch.